docs/excel/excel-add-ins-conditional-formatting.md

```diff
@@ -0,0 +1,305 @@
+# Apply conditional formatting to Excel ranges
+
+The Excel JavaScript Library provides APIs to apply conditional formatting to data ranges in your worksheets. This functionality makes large sets of data easy to visually parse. The formatting also dynamically updates based on changes within the range. The following articles provide detailed information about the full conditional formatting capabilities within Excel.
+
+-	[Add, change, or clear conditional formats](https://support.office.com/en-us/article/add-change-or-clear-conditional-formats-8a1cc355-b113-41b7-a483-58460332a1af)
+-	[Use formulas with conditional formatting](https://support.office.com/en-us/article/Use-formulas-with-conditional-formatting-FED60DFA-1D3F-4E13-9ECB-F1951FF89D7F)
+
+## Programmatic control of conditional formatting
+
+The `Range.conditionalFormats` property is a collection of [ConditionalFormat](https://docs.microsoft.com/javascript/api/excel/excel.conditionalformat) objects that apply to the range.  The `ConditionalFormat` object contains several properties defining the format to be applied based on the [ConditionalFormatType](https://docs.microsoft.com/javascript/api/excel/excel.conditionalformattype). 
```

Comment:
change "contains several properties defining..." to "contains several properties that define..." ?